src/langcheck/plot/_scatter.py

```diff
@@ -288,6 +292,17 @@ def update_figure(filter_generated_outputs, filter_reference_outputs,
                          x=metric_value.metric_name,
                          y=other_metric_value.metric_name,
                          hover_data=hover_data)
+        # Paint threshold
```

Comment:
```suggestion
        # Draw threshold
```
nit: I think "draw" is more natural than "paint" here